cola/themes.py

```diff
@@ -521,9 +527,22 @@ def style_sheet_flat(self):
             focus=focus,
         )
 
+    def style_sheet_custom(self, app_palette):
+        """Get custom style sheet.
+        File name is saved in variable self.name.
+        If user has deleted file, use default style"""
+
+        # check if path exists
+        filepath = resources.config_home('themes', self.name + '.qss')
+        if not os.path.exists(filepath):
+            return self.style_sheet_default(app_palette)
+
+        with open(filepath) as file:
+            return file.read()
```

Comment:
Maybe we should catch the exception here?

```
try:
    with open(filepath) as fp:
        return fp.read()
except (IOError, OSError) as err:
    sys.stderr.write('warning: unable to read custom theme %s: %s\n' % (filepath, err))
    return self.style_sheet_default(app_palette)
```

That way we'll get an error message if we're unable to read the custom config for some reason.

Reply:
added exception